motoman_driver/MotoPlus/Controller.c

```diff
@@ -92,68 +92,6 @@ void reportVersionInfoToController()
 #endif
 }
 
-// Verify most of the setup parameters of the robot controller.
-// Please note that some parameters cannot be checked, such as
-// the parameter(s) which enable this task to run.
-// Returns FALSE if a critical parameter is not set such that it
-// will prevent intialization.
-BOOL Ros_Controller_CheckSetup()
-{
-	int parameterValidationCode;
-
-	parameterValidationCode = ValidateMotoRosSetupParameters();
-	switch (parameterValidationCode)
-	{
-	case MOTOROS_SETUP_OK: return TRUE;
-
-	case MOTOROS_SETUP_RS0: 
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS Cfg: Set RS000=2", parameterValidationCode);
-		return TRUE;
-
-	case MOTOROS_SETUP_S2C541:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS Cfg: Set S2C541=0", parameterValidationCode);
-		return TRUE;
-
-	case MOTOROS_SETUP_S2C542:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS Cfg: Set S2C542=0", parameterValidationCode);
-		return TRUE;
-
-	case MOTOROS_SETUP_S2C1100:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS Cfg: Set S2C1100=1", parameterValidationCode);
-		return FALSE;
-
-	case MOTOROS_SETUP_S2C1103:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS Cfg: Set S2C1103=2", parameterValidationCode);
-		return FALSE;
-
-	case MOTOROS_SETUP_S2C1117:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS Cfg: Set S2C1117=1", parameterValidationCode);
-		return FALSE;
-
-	case MOTOROS_SETUP_S2C1119:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS Cfg: Set S2C1119=0 or 2", parameterValidationCode);
-		return TRUE;
-
-	case MOTOROS_SETUP_NotCompatibleWithPFL:
-	case MOTOROS_SETUP_NotCompatibleHCrobot:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS not compatible with HC10", parameterValidationCode);
-		return FALSE;
-
-#if (DX100)
-	case MOTOROS_SETUP_InvalidSdaConfiguration:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS: Reconfigure waist axis", parameterValidationCode);
-		return FALSE;
-#endif
-
-	//For all other error codes, please contact Yaskawa Motoman
-	//to have the MotoROS Runtime functionality enabled on your
-	//robot controller.
-	default:
-		mpSetAlarm(MOTOROS_SETUPERROR_ALARMCODE, "MotoROS: Controller cfg invalid", parameterValidationCode);
-		return FALSE;
-	}
-}
-

```

Comment:
This commit seems to remove all of the setup validation code, including the libraries, but I don't see it moved to somewhere else. How is the validation handled now?